lib/config.js

```diff
@@ -1,9 +1,20 @@
 const fp = require('fastify-plugin');
 
-module.exports = fp(async function (fastify) {
-  /* eslint-disable no-process-env */
-  fastify.decorate('config', {
-    port: process.env.PORT || 6666
-  });
-  /* eslint-enable no-process-env */
-});
+module.exports = fp(
+  async function (fastify) {
+    /* eslint-disable no-process-env */
+    fastify.decorate('config', {
+      port: process.env.PORT || 6666,
+      dynamo: {
+        accessKeyId: process.env.AWS_ACCESS_KEY_ID, // optional
+        secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY, // optional
```

Comment:
The AWS SDK should automatically pull from these vars, so you don't need to manually pass in auth.

Reply:
Reason why I did this is to have `region=us-west-2` as default, but ok. Let's remove this.